PixelToolkit.py

```diff
@@ -39,10 +39,11 @@
     wordlist_gen.add_argument("--max", type=int, help="Maximum keyword length", default=100)
 
     hash = subparsers.add_parser("hash", help="Text hashing")
-    hash.add_argument("--text", "-t", required=True, help="File or text to be hashed. Files will be hashed line by line.")
+    # TODO replace current input with file and text arguments.
+    hash.add_argument("--input", "-i", required=True, help="File or text to be hashed.")
     hash.add_argument("--algorithm", '-a', default="SHA256")
-    # User can now create file name to save the file.
-    hash.add_argument("--buf", "-b", required=True, help="Size of chunks", default=65536)
+    # TODO Implement automatic setting of buffer or remove buffer
+    hash.add_argument("--buf", "-b", required=False, help="Size of chunks", default="4096")
```

Comment:
why is `"4096"` a string? btw it's better to specify the type explicitly: `type=int`

Reply:
chaged